packages/react-keytips/src/components/Keytips/Keytips.test.tsx

```diff
@@ -116,4 +116,40 @@ describe('Keytips', () => {
     expect(screen.queryAllByRole('tooltip')).toHaveLength(0);
     expect(onExitKeytipsMode).toHaveBeenCalledTimes(1);
   });
+
+  it('should support different invoking event', async () => {
```

Comment:
In my opinion, the current test focuses more on the implementation rather than the behavior it's supposed to assess. I recommend substituting it with two Playwright tests that specifically evaluate the functionality in both keydown and keyup modes. What do you think?

Reply:
not sure if there are two tests needed, keydown is already covered by basic testing in playwright, it would be a duplication. Although I can add the keyup test to test it in playwright, instead of testing-library, as it's now

Reply:
Added, I don't mind having both things tested, but having only keyup test cases should be enough, as keydown tested by other test cases.